### ISSUE TITLE
Keyboard shortcuts to navigate back and forward (macOS)

### ISSUE THREAD

**Issue by breyed** [codemerx/CodemerxDecompile#112]
On macOS (Apple silicon), there are no keyboard shortcuts to navigate back and forward.

**Comment by atdimitrov** [comment]
Hi @breyed. Cmd+- and Cmd+Shift+- should work as back and forward, respectively. Does that work for you? I've tested it and it's working on my Apple Silicon Mac.

**Comment by breyed** [comment]
Yes. I didn’t think to try those keys. I also missed that they are discoverable via the Navigate menu’s display of keyboard shortcuts.

**Comment by breyed** [comment]
FWIW, while Visual Studio uses to Ctrl+- for back, Safari and Chrome use Cmd+[, and Xcode uses Ctrl+Cmd+Left.

**Comment by atdimitrov** [comment]
For the shortcut scheme, I think it makes the most sense to follow IDEs as CodemerxDecompile kind of looks like one. It certainly has the navigation of one. As it is a .NET-oriented tool I wouldn't look into Xcode as normally developers are not using it for .NET development. So for MacOS we have:
- Visual Studio Code uses Ctrl+-, while it uses Cmd+Click to navigate to a definition. The mixed usage of Cmd and Ctrl is a bit weird in my opinion, but somewhat understandable as Ctrl+Click has a special meaning in MacOS.
- Visual Studio for Mac seems to be using Ctrl+Cmd+- for back. Not sure what it uses to navigate to definition though. I assume it is the same Cmd+Click.
- Rider, as a popular third-party option, has many shortcut schemes. CodemerxDecompile matches the Rider's ReSharper (macOS) scheme. Few other schemes, that are based on VS or VS code do use the same Ctrl+-.

TL;DR; I can understand the frustration of having to recall different key combinations in different apps for the same basic action. As fully-fledged custom key binding configuration is out of reach for now I think it is going to be beneficial to add Ctrl+- and Ctrl+Shift+- as a second option alongside with the Cmd counterparts. That should solve the issue for most users.

**Comment by breyed** [comment]
Actually, CodemerxDecompile already supports key binding configuration on macOS. Since navigate forward and back are in the menu, a user can assign a custom shortcut via [App Shortcuts](https://support.apple.com/en-gb/guide/mac-help/mchlp2271/mac).
